Make Data Count Test failing on Develop

Seeing one test failure on the develop branch (ex. https://jenkins.dataverse.org/job/IQSS-dataverse-develop/794/testReport/junit/edu.harvard.iq.dataverse.api/MakeDataCountApiIT/testMakeDataCountGetMetric/)

> Error Message
> JSON path message doesn't match.
> Expected: MetricType must be one of these values: [viewsTotal, viewsTotalRegular, viewsTotalMachine, viewsUnique, viewsUniqueRegular, viewsUniqueMachine, downloadsTotal, downloadsTotalRegular, downloadsTotalMachine, downloadsUnique, downloadsUniqueRegular, downloadsUniqueMachine, citations].
>   Actual: MetricType must be one of these values: [viewsTotalMachine + viewsTotalRegular, viewsTotalRegular, viewsTotalMachine, viewsUniqueMachine + viewsUniqueRegular, viewsUniqueRegular, viewsUniqueMachine, downloadsTotalMachine + downloadsTotalRegular, downloadsTotalRegular, downloadsTotalMachine, downloadsUniqueMachine + downloadsUniqueRegular, downloadsUniqueRegular, downloadsUniqueMachine, citations].
> Stacktrace
> java.lang.AssertionError: 
> JSON path message doesn't match.
> Expected: MetricType must be one of these values: [viewsTotal, viewsTotalRegular, viewsTotalMachine, viewsUnique, viewsUniqueRegular, viewsUniqueMachine, downloadsTotal, downloadsTotalRegular, downloadsTotalMachine, downloadsUnique, downloadsUniqueRegular, downloadsUniqueMachine, citations].
>   Actual: MetricType must be one of these values: [viewsTotalMachine + viewsTotalRegular, viewsTotalRegular, viewsTotalMachine, viewsUniqueMachine + viewsUniqueRegular, viewsUniqueRegular, viewsUniqueMachine, downloadsTotalMachine + downloadsTotalRegular, downloadsTotalRegular, downloadsTotalMachine, downloadsUniqueMachine + downloadsUniqueRegular, downloadsUniqueRegular, downloadsUniqueMachine, citations].
> 
> 	at edu.harvard.iq.dataverse.api.MakeDataCountApiIT.testMakeDataCountGetMetric(MakeDataCountApiIT.java:71)
> Standard Output
> {"status":"OK","data":{"user":{"id":53,"userName":"user04e09346"},"authenticatedUser":{"id":53,"identifier":"@user04e09346","displayName":"user04e09346 user04e09346","firstName":"user04e09346","lastName":"user04e09346","email":"user04e09346@mailinator.com","superuser":false,"deactivated":false,"persistentUserId":"user04e09346","createdTime":"2021-05-12T22:15:37Z","lastLoginTime":"2021-05-12T22:15:37Z","authenticationProviderId":"builtin"},"apiToken":"441f696f-d5ac-461b-9e57-53dd8d93d29c"}}
> {"status":"OK","data":{"id":38,"alias":"dv11a47b93","name":"dv11a47b93","dataverseContacts":[{"displayOrder":0,"contactEmail":"616aeff7@mailinator.com"}],"permissionRoot":true,"dataverseType":"UNCATEGORIZED","ownerId":1,"creationDate":"2021-05-12T22:15:37Z"}}
> {"status":"OK","data":{"id":39,"persistentId":"doi:10.5072/FK2/KV2VPO"}}
> /api/admin/makeDataCount/39/addUsageMetricsFromSushiReport?reportOnDisk=/tmp/sushi_sample_logs.json{"status":"OK","data":{"message":"Dummy Data has been added to dataset 39"}}
> metric: junk
> {"status":"ERROR","message":"MetricType must be one of these values: [viewsTotalMachine + viewsTotalRegular, viewsTotalRegular, viewsTotalMachine, viewsUniqueMachine + viewsUniqueRegular, viewsUniqueRegular, viewsUniqueMachine, downloadsTotalMachine + downloadsTotalRegular, downloadsTotalRegular, downloadsTotalMachine, downloadsUniqueMachine + downloadsUniqueRegular, downloadsUniqueRegular, downloadsUniqueMachine, citations]."}
> Standard Error
> May 12, 2021 10:15:37 PM edu.harvard.iq.dataverse.api.UtilIT createRandomUser
> INFO: Creating random test user user04e09346
> May 12, 2021 10:15:37 PM edu.harvard.iq.dataverse.api.UtilIT getUsernameFromResponse
> INFO: Username found in create user response: user04e09346
> May 12, 2021 10:15:37 PM edu.harvard.iq.dataverse.api.UtilIT getApiTokenFromResponse
> INFO: API token found in create user response: 441f696f-d5ac-461b-9e57-53dd8d93d29c
> May 12, 2021 10:15:37 PM edu.harvard.iq.dataverse.api.UtilIT getAliasFromResponse
> INFO: Alias found in create dataverse response: dv11a47b93
> May 12, 2021 10:15:38 PM edu.harvard.iq.dataverse.api.UtilIT getDatasetIdFromResponse
> INFO: Id found in create dataset response: 39
>

--- a/src/main/java/edu/harvard/iq/dataverse/makedatacount/MakeDataCountUtil.java
+++ b/src/main/java/edu/harvard/iq/dataverse/makedatacount/MakeDataCountUtil.java
@@ -87,7 +87,15 @@ public class MakeDataCountUtil {
                     }
                 }
             }
-            throw new IllegalArgumentException("MetricType must be one of these values: " + Arrays.asList(MetricType.values()) + ".");
+            throw new IllegalArgumentException("MetricType must be one of these values: " + getMetricNameList() + ".");
+        }
+        
+        private static List<String> getMetricNameList() {
+           ArrayList<String> names = new ArrayList<String>();
+           for(MetricType mt: MetricType.values()) {
+               names.add(mt.text);
+           }
+            return names;
         }
 
         @Override

